component/admin/language/en-GB.com_jevents.ini

```diff
@@ -1435,7 +1435,7 @@ JEV_SHOW_TIMEZONE_DESC="When enabled allows the event editor to specify the time
 JEV_CSV_EXPORT="Export results as CSV file"
 JEV_CSV_EXPORT_DESC="Allows exporting of event data in CSV file for chosen columns"
 
-JEV_CHECK_OVERLAPPING_REPEATS="WARNING: You have created a series of repeating events that overlap each other.  This is probably a mistake please check your event dates and repeat pattern"
+JEV_CHECK_OVERLAPPING_REPEATS="WARNING: You have created a series of repeating events that overlap each other. This is probably a mistake please check your event dates and repeat pattern"
```

Comment:
In standard english typing you always have 2 spaces after a full stop, 1 after a comma ;)